app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2 class="govuk-heading-m"><%= t('.page_heading') %></h1>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: resource, scope: resource_name, url: user_session_path(resource_name)) do |f| %>
+      <div class="field">
+        <%= f.govuk_email_field :email, label: { text: t('.email') }, autofocus: true, autocomplete: 'email' %>
+      </div>
+
+      <div class="field">
+        <%= f.govuk_password_field :password, label: { text: t('.password')}, autocomplete: 'current-password' %>
+      </div>
+
+      <div class="actions">
```

Comment:
Minor: is this additional element `div.actions` required considering the form helper will render the relevant element with default style and spacing?

Reply:
👍 have removed